cmd/skopeo/utils_test.go

```diff
@@ -42,10 +44,16 @@ func fakeImageOptions(t *testing.T, flagPrefix string, globalFlags []string, cmd
 
 func TestImageOptionsNewSystemContext(t *testing.T) {
 	// Default state
+
+	// Make sure when XDG_RUNTIME_DIR is set, defaults to "$XDG_RUNTIME_DIR/containers/auth.json"
+	runtimeDir := "/run/test/0"
+	os.Setenv("XDG_RUNTIME_DIR", runtimeDir)
```

Comment:
See how `signing_test.go` ensures the test overrides don’t leak into other tests.

Or even https://github.com/containers/image/blob/d13a90f787a7d3cd58e6f13c7cd1f2307b74a1ef/pkg/blobinfocache/default_test.go#L24-L33 .

Reply:
Fixed